ui/desktop/electron-app/src/spawn-promise.js

```diff
@@ -0,0 +1,46 @@
+const { spawn } = require('child_process');
+
+// Verify json
+const isJSON = (data) => {
```

Comment:
It seems like this method is not exactly "is JSON".  It's more "is an object in object or JSON string form".

Reply:
Updated to `jsonify` function that returns data as json, is possible.